neo/Ledger/MemoryPool.cs

```diff
@@ -337,18 +342,30 @@ internal void InvalidateVerifiedTransactions()
         // Note: this must only be called from a single thread (the Blockchain actor)
         internal void UpdatePoolForBlockPersisted(Block block, Snapshot snapshot)
         {
+            LoadPolicy(snapshot);
```

Comment:
Simple change indeed.

Reply:
Erik, I suggest that `LoadPolicy` returns a boolean, indicating the need to reverify transactions. It's better to avoid yet another variable.